app/src/main/java/com/kickstarter/viewmodels/CreatorDashboardHeaderHolderViewModel.java

```diff
@@ -0,0 +1,128 @@
+    void projectViewClicked();
+  }
+
+  interface Outputs {
+    /* string number with the percentage of a projects funding */
+    Observable<String> percentageFunded();
+
+    /* localized count of number of backers */
+    Observable<String> projectBackersCountText();
+
+    /* current project's name */
+    Observable<String> projectNameTextViewText();
+
+    /* project that is currently being viewed */
+    Observable<Project> currentProject();
+
+    /* time remaining for latest project (no units) */
+    Observable<String> timeRemainingText();
+
+    /* call when button is clicked to view individual project page */
+    Observable<Pair<Project, RefTag>> startProjectActivity();
+  }
+
+  final class ViewModel extends ActivityViewModel<CreatorDashboardHeaderViewHolder> implements
+    Inputs, Outputs {
+
+    public ViewModel(final @NonNull Environment environment) {
+      super(environment);
+
+      this.currentProject =  projectAndStats
+        .map(PairUtils::first);
+
+      this.percentageFunded = projectAndStats
+        .map(PairUtils::first)
+        .map(Project::percentageFunded)
+        .map(NumberUtils::flooredPercentage)
+        .compose(bindToLifecycle());
+
+      this.projectBackersCountText = projectAndStats
+        .map(PairUtils::first)
+        .map(Project::backersCount)
+        .map(NumberUtils::format)
+        .compose(bindToLifecycle());
+
+      this.projectNameTextViewText = projectAndStats
+        .map(PairUtils::first)
+        .map(Project::name)
+        .distinctUntilChanged()
+        .compose(bindToLifecycle());
+
+      this.timeRemainingText = projectAndStats
+        .map(PairUtils::first)
+        .map(ProjectUtils::deadlineCountdownValue)
+        .map(NumberUtils::format);
+
+      this.startProjectActivity = this.currentProject()
+        .compose(Transformers.takeWhen(projectViewClicked))
+        .map(p -> Pair.create(p, RefTag.dashboard()));
+    }
+
+
+    public final Inputs inputs = this;
+    public final Outputs outputs = this;
+
+    private final PublishSubject<Pair<Project, ProjectStatsEnvelope>> projectAndStats = PublishSubject.create();
+    private final PublishSubject<Void> projectViewClicked = PublishSubject.create();
+
+    private final Observable<String> percentageFunded;
+    private final Observable<Project> currentProject;
+    private final Observable<String> projectBackersCountText;
+    private final Observable<String> projectNameTextViewText;
+    private final Observable<Pair<Project, RefTag>> startProjectActivity;
+    private final Observable<String> timeRemainingText;
+
+    @Override
+    public void projectViewClicked() {
+      this.projectViewClicked.onNext(null);
+    }
+
+    @Override
+    public void projectAndStats(final @NonNull Project project, final @NonNull ProjectStatsEnvelope ProjectStatsEnvelope) {
+      this.projectAndStats.onNext(Pair.create(project, ProjectStatsEnvelope));
```

Comment:
since we're just making a new pair here, we can change the `projectAndStats` input type above on line 24 to just be a `Pair<Project, Stats>`, and input it directly from the [ViewHolder](https://github.com/kickstarter/android-oss/pull/130/files#diff-f8896b2518d63fc0f3f5fbad2e11e15eR102) since we have the 🍐 there